.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -1,73 +1,64 @@
-name: "🐛 Bug report"
+name: "🕷️ Bug report"
 description: Report errors or unexpected behavior
-labels: 
+labels:
 - Issue-Bug
 - Needs-Triage
 body:
 - type: markdown
   attributes:
-    value: |
-      Please make sure to [search for existing issues](https://github.com/microsoft/PowerToys/issues) before filing a new one!
+    value: Please make sure to [search for existing issues](https://github.com/microsoft/PowerToys/issues) before filing a new one!
 - type: input
   attributes:
     label: Microsoft PowerToys version
-    placeholder: |
-      "0.53.0"
-    description: |
-      Hover over system tray icon or look at Settings
+    placeholder: 0.63.0
+    description: Hover over system tray icon or look at Settings
   validations:
     required: true
 
-- type: checkboxes
+- type: dropdown
   attributes:
     label: Running as admin
     description: Are you running PowerToys as Admin?
     options:
-      - label: "Yes"
+      - "Yes"
+      - "No"
 
 - type: dropdown
   attributes:
     label: Area(s) with issue?
-    description: What things had an issue?  Check all that apply.
+    description: What things had an issue? Check all that apply.
     multiple: true
     options:
       - General
       - Always on Top
       - Awake
       - ColorPicker
-      - Developer file preview
       - FancyZones
       - FancyZones Editor 
       - Image Resizer
+      - Installer
       - Keyboard Manager
-      - MD Preview
       - Mouse Utilities
-      - PDF Preview
-      - PDF Thumbnail
-      - G-code Preview
-      - G-code Thumbnail
       - PowerRename
       - PowerToys Run
+      - Preview Pane (Explorer)
       - Quick Accent
       - Screen ruler
-      - Shortcut Guide
-      - STL Thumbnail
-      - SVG Preview
-      - SVG Thumbnail
       - Settings
+      - Shortcut Guide
+      - System tray interaction
       - TextExtractor
+      - Thumbnail preview (Explorer)
       - Video Conference Mute
       - Welcome / PowerToys Tour window
-      - System tray interaction
-      - Installer
   validations:
     required: true
 
 - type: textarea
   attributes:
     label: Steps to reproduce
-    description: We highly suggest including screenshots and a bug report log (System tray->Report bug).
-    placeholder: Tell us the steps required to trigger your bug.
+    description: We highly suggest including screenshots and a bug report log (System tray > Report bug).
```

It was to form an arrow, 
```suggestion
    description: We highly suggest including screenshots and a bug report log (System tray → Report bug).
```

---

Specific:
https://learn.microsoft.com/style-guide/procedures-instructions/describing-interactions-with-ui#:~:text=Use%20a%20greater%2Dthan%20symbol%20(%3E)%20to%20separate%20sequential%20steps.